src/go/k8s/api/redpanda/v1alpha2/topic_types.go

```diff
@@ -0,0 +1,384 @@
+// Copyright 2021-2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package v1alpha2
```

Comment:
This file is just moved and has a package rename, it shows as a diff rather than a move due to the `topic_types.go` file in `v1alpha1` still existing with a type definition to make `v1alpha1.Topic[List]` the same as `v1alpha2`.